ai-service/tests/conftest.py

```
@@ -1,48 +1,50 @@
-    client = chromadb.PersistentClient(path=chroma_path)
-    return client.get_or_create_collection(collection_name)
-
-
-@pytest.fixture(scope="module")
-def db_test_collection(request: pytest.FixtureRequest) -> chromadb.Collection:
-    module = getattr(request, "module")
-    module_name = getattr(module, "__name__")
-    collection_name: str = f"{module_name}"
-    return create_db_test_collection(collection_name)
+@pytest.fixture(scope="session", autouse=True)
+def setup_services_for_integration_tests():
```

Ruff (0.12.2)

Missing return type annotation for public function `setup_services_for_integration_tests`

(ANN201)